### PR TITLE
Small tweaks

### PR DESCRIPTION
**Pull request by ColonelGerdauf**
removed redundancies
corrected a faulty rule param
corrected group orientations [incomplete]

### CODE REVIEW

**Comment by ColonelGerdauf** [comment]
Finished the cleanup of rules. Found a few more redundancies.

When writing or modifying a rule; keep this order in mind

> objects
> conditions
> classes
> id's

**Comment by the-j0k3r** [comment]
This is very well fine and dandy contribution but ignores several things.

Add version patch, this is internal development only (please dont)
No perfectionist ran on this selectors are now in separate lines (please do)

No idea how many breakages this will introduce, given the sheer insanity that we try to accommodate in a single style for a zillion pages. 

@AfroThundr3007730 for final decision.

File: wikipedia-dark.user.css [hunk 1]
```diff
@@ -1490,13 +1840,19 @@ regexp("https?:\/\/wikiless\.org\/.*$") {
   .mwe-popups .mwe-popups-container .mwe-popups-thumbnail {
     background-color: var(--white) !important;
   }
-  span[style="color:black"], span[style*="color: black"],
-  span[style*="color:black"], td[style*="color: black"],
-  td[style*="color: #2C2C2C"], .mwe-popups-container a {
+  span[style="color:black"],
+  span[style*="color: black"],
+  span[style*="color:black"],
```
Comment (the-j0k3r) [inline wikipedia-dark.user.css:1845]: one of these can be removed, span[style*="color:black"] will match span[style*="color:black"] so keep the span[style*="color:black"]

**Comment by AfroThundr3007730** [comment]
From what I can see after reviewing this, you split the selectors to be one per line, then sorted them by type. It looks like there are no functional changes, aside from removing a redundant selector above.

To reduce the size of the CSS, we typically run `perfectionist` to group the selectors, which your first commit basically reverses. If we wanted to go with a one selector per line format, sorting them (as per your second commit) makes sense.

So what advantages do we gain from this expanded and sorted format? If the goal is increased legibility and maintainability, I would agree on that point. The 10% line count increase doesn't actually increase the file size much (2KB).

Still, there's not much reason to do a reformatting on its own, I think, unless this is meant to be followed up with further changes. @ColonelGerdauf and @the-j0k3r thoughts?

**Comment by ColonelGerdauf** [comment]
The reason for splitting the selectors to one per line is that it makes it significantly easier to review the existing selectors and seek problematic or redundant entries.

As noted by Joker, the current code base as it was was a nightmare in terms of legacy code with immense difficulties in removing outdated entries. This now should make it easier for people to troubleshoot.

**Comment by ColonelGerdauf** [comment]
Another benefit of separating the selectors to line by line is that it makes the process simple of merging selectors with the same declarations. It might make the code look a bit ugly, but in my opinion it is much easier than finding the same list of declarations written two or three times.

**Pull request merged**